src/common/executor.ts

```diff
@@ -35,6 +37,13 @@ export class Executor {
 
             let stderr: string = "";
             let stdOutput: string = "";
+            const envVars = Executor.getEnvFromConfig();
+            if (envVars) {
+                options = options || {};
+                const processEnvs = Executor.mergeEnvs(envVars, process.env);
+                options.env = Executor.mergeEnvs(options.env, processEnvs);

```

Comment:
Just confirm, the priority order of envs is: `options.env` > envs from config > `process.env`?

Reply:
Yes. options.env is passed by caller with intend. Envs from config is an adds-on to the vs code process.